README.md

```diff
@@ -14,7 +14,8 @@
 ## Install
 
 ```bash
-npm i tsdown
+npm i -D tsdown # Project install
+npm i -g tsdown # or Global install
```

Comment:
Should we be encouraging people to install tsdown globally though?

```suggestion
```

Reply:
You're right. Global install is not recommended.